README.md

```diff
@@ -124,8 +121,16 @@ VS Code
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[Online Booking]**
-- [ ] **[Updates]**
+- [ ] **[Navigation]**
+- [ ] **[Colourful Pages]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [Detailed Document](https://github.com/ehmaddd/mobile-first-portfolio/edit/main/README.md).
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you also add your issues page link here which you can see in the readme template of microverse :+1: to make your readme file more professional :+1: